src/app/products_view/product_availability_details/product_availability_details_test.ts

```diff
@@ -0,0 +1,88 @@
+/* tslint:disable:no-unused-variable */
```

Comment:
Why do you need to disable this?

Reply:
Its added to avoid lint errors when harness is having issues and tests are failing. Now updated, hence not required.